src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+public class Todo extends Task {
```

Comment:
Consider naming in Pascal case to conform to the coding standard!

Reply:
Thanks for reminding on this! I've edited the class name in a commit further downstream :)